src/chrome/handleWork.js

```diff
@@ -1,9 +1,10 @@
 import listenStoreChanges from '../listenStoreChanges';
 import { workFinished } from '../actions';
-import { mainAudioPlayer, playSound } from '../playSound';
+import { playSoundOptions, stopSound } from '../playSound';
 
 const handleWork = (store, chrome) => {
   let workTabId = null;
+  window.audioPlayer = null;
```

Comment:
I'd like to avoid this kind of global variable in general. Is it possible to make it module-local (like `workTabId`)? We shouldn't be sharing state between modules; instead each module should have its own state and react to redux events.